upgrade-ls

```diff
@@ -91,8 +91,30 @@ if [[ "$UBUNTU_VERSION" == "20.04" ]]; then
       mv -v "/usr/lib/node_modules/lamassu-server" "/usr/lib/node_modules/${BKP_NAME}" >> ${LOG_FILE} 2>&1
   fi
 
+  if ! [ -f ./verify ]
+  then
+    decho "Downloading verifier..."
+    curl -O https://raw.githubusercontent.com/lamassu/lamassu-install/grand-gilgamesh/verify 
+  fi
+
+  decho "Downloading update package..."
+  curl -o /tmp/lamassu-server-v8.1.0.tar.gz -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/lamassu-server-v8.1.0.tar.gz
+
+  curl -o /tmp/sig1.asc -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/sig1.asc
+  curl -o /tmp/sig2.asc -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/sig2.asc
+
+  decho "Verifying update package..."
+  lamassu-verify /tmp/lamassu-server-v8.1.0.tar.gz /tmp/sig1.asc /tmp/sig2.asc
```

Comment:
Where does this `lamassu-verify` come from? I assume it's the downloaded file, but that one is named `verify`

Reply:
Yes. Original idea was to put verify binary into `/usr/bin` as `lamassu-verify`, but I dropped that idea. Fixed this now, and fixed few more things